app/styles/organisms/carousel.less

```diff
@@ -0,0 +1,167 @@
+.oss-carousel {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  align-items: center;
+  gap: var(--spacing-px-12);
+  position: relative;
+
+  .page-btn {
+    padding: var(--spacing-px-3);
+    color: var(--color-gray-200);
+    font-size: 8px;
+
+    &:hover,
+    &--active {
+      color: var(--color-gray-500);
+    }
+  }
+
+  .carousel-control {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    z-index: 1;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: var(--spacing-px-24);
+    padding: 0;
+    color: var(--color-gray-500);
+    text-align: center;
+    transition: opacity .15s ease;
+
+    &--left {
+      left: 0;
+    }
+
+    &--right {
+      left: unset;
+      right: 0;
+    }
+  }
+
+  .page-container {
+    display: block;
+    position: relative;
+    overflow: hidden;
+    width: 100%;
+
+    .page {
+      width: 100%;
+      margin-right: -100%;
+      display: none;
+      float: left;
+      position: relative;
+      transition: transform .5s ease-in-out;
```

Comment:
Sync with design for this duration of animation?

Reply:
Nope, I replicated bootstrap's timings for this one as we will need to replace existing carousels in the software.